jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/Cloudformation.kt

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudformation
+
+import software.amazon.awssdk.services.cloudformation.CloudFormationClient
+import software.amazon.awssdk.services.cloudformation.model.CloudFormationException
+import software.amazon.awssdk.services.cloudformation.model.DescribeStacksRequest
+import software.amazon.awssdk.services.cloudformation.model.StackStatus
+import software.aws.toolkits.resources.message
+import java.util.concurrent.TimeoutException
+
+fun CloudFormationClient.waitForStackDeletionComplete(stackName: String, maxAttempts: Int, delay: Int) {
```

Comment:
delay should be a `Duration` not an int

Reply:
Kinda borrowed the Int from Java SDK. Our current waiter is configuring the delay as an int in second. Do we really need the delay in nano level?

https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/waiters/FixedDelayStrategy.java#L45

Reply:
Java 6 doesn't have the the `Duration` object, it was only introduced in Java 8 - if we're representing a duration (we are) then we should make that obvious through the type system.

Reply:
Done.